hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/Region.java

```diff
@@ -476,6 +476,7 @@ boolean checkAndRowMutate(byte [] row, Filter filter, TimeRange timeRange,
 
   /**
    * Do a get based on the get parameter.
+   * Only used for test purpose.
```

Comment:
What do you mean by this? That 'get' is for test only? Do you want to say more in this javadoc?

Reply:
I'll delete this change, and make HRegion.get() be used in CP.